### PR TITLE
Add BigNumber.js support

### PR DESCRIPTION
**Pull request by gre**
types ported from https://github.com/MikeMcl/bignumber.js/blob/master/bignumber.d.ts and required some manual adaptation (static fields, function style call of BigNumber,..) which i think was not covered by the other PR
test code coming from README of bignumber.js https://www.npmjs.com/package/bignumber.js

### CODE REVIEW

File: definitions/npm/bignumber.js_v7.x.x/test_bignumber.js_v7.x.x.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// @flow
```
Comment (AndrewSouthpaw) [inline definitions/npm/bignumber.js_v7.x.x/test_bignumber.js_v7.x.x.js]: Put test file under the flow versioned folder.

File: definitions/npm/bignumber.js_v7.x.x/test_bignumber.js_v7.x.x.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { BigNumber } from "bignumber.js";
+
+const x = new BigNumber(123.4567);
+const y = BigNumber("123456.7e-3");
+const z = new BigNumber(x);
+x.isEqualTo(y) && y.isEqualTo(z) && x.isEqualTo(z);
+const a = new BigNumber(1011, 2);
+const b = new BigNumber("zz.9", 36);
+const c = x.plus(y);
+x.minus(y);
```
Comment (AndrewSouthpaw) [inline definitions/npm/bignumber.js_v7.x.x/test_bignumber.js_v7.x.x.js]: Break apart tests using `describe` and `it`.

File: definitions/npm/bignumber.js_v7.x.x/flow_v0.55.x-/bignumber.js_v7.x.x.js [hunk 3]
```diff
@@ -0,0 +1,1685 @@
+declare module "bignumber.js" {
```
Comment (AndrewSouthpaw) [inline definitions/npm/bignumber.js_v7.x.x/flow_v0.55.x-/bignumber.js_v7.x.x.js]: Credit the original file, so we know from where it was ported.

**Comment by gre** [comment]
@AndrewSouthpaw thanks done

**Pull request merged**